src/modules/roc_audio/resampler_reader.cpp

```diff
@@ -32,6 +34,9 @@ ResamplerReader::ResamplerReader(IReader& reader,
     if (!init_frames_(buffer_pool)) {
         return;
     }
+    if(frame_size_ == 0) {
+        return;
+    }
```

Comment:
It's better to check frame_size_ before calling init_frames_, because init_frames_ uses frame_size_.

Reply:
done.